Gemfile

```diff
@@ -2,6 +2,7 @@ source "https://rubygems.org"
 
 gem "fastlane", '~> 2.144.0'
 gem "badge", '~> 0.10.0'
+gem "cocoapods"
```

Comment:
What do you think of adding version of "cocoapods" like `gem "cocoapods", '~>1.9.1'` to work on the same environment?

Reply:
I wouldn't do it as long as it's not really required, IMO it's better to use the latest version. That said, CocoaPods version is still defined in `Gemfile.lock`: https://github.com/ProteGO-app/ios/pull/56/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R40
So, whoever does `bundle install` will end up with version `1.9.1`. It can't be bumped up accidentally.

Reply:
[Przejdę na polski, tak jak w reszcie PRów w tym projekcie:) ]

Proponowałbym jednak zaszycie konkretnego numeru cocoapodsów, aby nasze buildy były reprodukowalne.

Rozumiem fakt, ze dobrze jest pracować na najnowszych wersjach, jednak wolałbym spełniać ten fakt często podbijając ręcznie numery wersji, mając jednocześnie pełną kontrolę na środowiskiem w jakim budujemy wersje aplikacji.

```Relevant attributes of the build environment would usually include dependencies and their versions, build configuration flags and environment variables as far as they are used by the build system (eg. the locale). It is preferable to reduce this set of attributes.```
https://reproducible-builds.org/docs/definition/


Dodatkowo generalnie staramy trzymać się zasady blokowania wersji zależności, bo dzięki temu mamy lepszą kontrolę ich bezpieczeństwem:

```All third party components used by the mobile app, such as libraries and frameworks, are identified, and checked for known vulnerabilities.```

https://mobile-security.gitbook.io/masvs/security-requirements/0x12-v7-code_quality_and_build_setting_requirements

Reply:
Jasne, rozumiem, ma to sens. Dostosowałem mój PR. ✅ :)